pkg/skaffold/schema/latest/config.go

```diff
@@ -631,6 +631,9 @@ type DockerArtifact struct {
 	// CacheFrom lists the Docker images used as cache sources.
 	// For example: `["golang:1.10.1-alpine3.7", "alpine:3.7"]`.
 	CacheFrom []string `yaml:"cacheFrom,omitempty"`
+	
+	// pass in --no-cache to docker build to prevent caching
```

Comment:
I think this comment needs to be in the form 

`NoCache blah blah blah.` otherwise the tests will complain :sweat_smile:

Reply:
Completed